### PR TITLE
Standardized the class headers

### PR DESCRIPTION
**Pull request by evrifaessa**
This PR standardizes the class headers at the top of the files. For reference, I used the one in `index.php`. If you want to make changes on the text, it's a very easy change and I can do it within this PR.

Also removed the redundant phpDoc blocks above `setDi` and `getDi` methods where they were left over.

### CODE REVIEW

File: cspell.json [hunk 1]
```diff
@@ -336,7 +336,8 @@
         "Autoloader",
         "autoloader",
         "qwertyuiopasdfghjklzxccvbnm",
-        "QWERTYUIOPASDFGHJKLZXCVBNM"
+        "QWERTYUIOPASDFGHJKLZXCVBNM",
+        "Servicecentovacast"
```
Comment (jaapmarcus) [inline cspell.json:340]: Servicecentovacast has been dropped so why not remove it from 

https://github.com/FOSSBilling/FOSSBilling/pull/1212/files#diff-8ec5bae185dc9af5b5bfdcd5f1dcbee62f18525ee488996d52b22729f4076ef0

**Review by jaapmarcus — APPROVED** [review]

**Comment by admdly** [comment]
Nice one - this was also on my todo list. :)

If we’re updating the file header, it might be a good time to also standardise it to a recommended form (see [here](https://apache.org/foundation/license-faq.html#Apply-My-Software)). 

I’d be happy to go with with either the full or short header, but the shorter version is better and more concise in my opinion. That is:

```
/**
 * Copyright 2022-2023 FOSSBilling
 * Copyright 2011-2021 BoxBilling, Inc. 
 * SPDX-License-Identifier: Apache-2.0
 *
 * @copyright FOSSBilling (https://www.fossbilling.org)
 * @license http://www.apache.org/licenses/LICENSE-2.0 Apache-2.0
 */
```

File: .php-cs-fixer.dist.php [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-
 /**
- * FOSSBilling
+ * FOSSBilling.
  *
  * @copyright FOSSBilling (https://www.fossbilling.org)
  * @license   Apache-2.0
  *
- * This file may contain code previously used in the BoxBilling project.
+ * Copyright FOSSBilling 2022
```
Comment (BelleNottelling) [inline .php-cs-fixer.dist.php:8]: 2022?
It is 2023 now ;)

**Comment by BelleNottelling** [comment]
> Nice one - this was also on my todo list. 🙂
> 
> If we’re updating the file header, it might be a good time to also standardise it to a recommended form (see [here](https://apache.org/foundation/license-faq.html#Apply-My-Software)).
> 
> I’d be happy to go with with either the full or short header, but the shorter version is better and more concise in my opinion. That is:
> 
> ```
> /**
>  * Copyright 2022-2023 FOSSBilling
>  * Copyright 2011-2021 BoxBilling, Inc. 
>  * SPDX-License-Identifier: Apache-2.0
>  *
>  * @copyright FOSSBilling (https://www.fossbilling.org)
>  * @license http://www.apache.org/licenses/LICENSE-2.0 Apache-2.0
>  */
> ```

I like this one more. Is there a way we can have PHP-CS-Fixer only apply this to files that are actually from BoxBilling?
IE: Would it be practical for our new classes to only included a copyright for FOSSBilling?

**Comment by evrifaessa** [comment]
@BelleNottelling @admdly done. :)

> Is there a way we can have PHP-CS-Fixer only apply this to files that are actually from BoxBilling?

I don't really know if we can get it to differentiate.

**Comment by BelleNottelling** [comment]
> I don't really know if we can get it to differentiate.

The only way I could think of is if it let us do different headers depending on the path. It's not that important, though

**Review by BelleNottelling — APPROVED** [review]
LGTM, thanks :)

**Review by admdly — APPROVED** [review]

**Comment by admdly** [comment]
Merged main branch to resolve conflict.

**Pull request merged**